Dockerfile

```diff
@@ -53,13 +53,14 @@ RUN cat ./frontend/.env
 # Backwards compatibility, move plugin folder to only copy matching plugins.
 RUN mv plugins plugins-old || true
 
+# Copy a .plugins folder if it is there to ./plugins, otherwise create an empty one.
+COPY ./.plugi*s ./plugins
```

Comment:
Is there a reason for using `.plugi*s`? I am sorry I don't understand.

Reply:
Thanks.

I added further explanation of this... let me know if it makes sense now?

Reply:
Cool. Thanks.